src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -73,4 +73,24 @@ export default class ReimbursementRequestService {
 
     return vendor.vendorId;
   }
+
+  /**
+   * Service function to create an expense type in our database
+   * @param name The name of the expense type
+   * @param code the expense type's code
+   * @param allowed whether or not this expense type is allowed
```

Comment:
ok but what does this actually mean, cause this is confusing to me

Reply:
i think it means whether or not people are allowed to use this expense type? I honestly dont know either it was just a field in the frontend form